ui/jobs/jobs_config.js

```diff
@@ -41,6 +41,15 @@ UserConfig.registerOptions('jobs', {
       type: 'float',
       default: 0.5,
     },
+    {
+      id: 'PlayCountdownSound',
+      name: {
+        en: 'Enable countdown alarm sound',
```

Comment:
I think use "Alert" instead of "Alarm" would be more consistent ?

Reply:
That, or "notification". (I say this because `raidboss` already has meanings for "info", "alert", and "alarm", and it might be better to avoid any potential confusion there.)